model_analyzer/model_manager.py

```diff
@@ -136,6 +144,23 @@ def _get_triton_server_flags(self, models):
                     f"Triton server flags must be the same for all models to run concurrently"
                 )
 
+    def _check_for_ensemble_model_incompatability(
+            self, models: List[ConfigModelProfileSpec]) -> None:
+        for model in models:
+            model_config = ModelConfig.create_from_profile_spec(
+                model, self._config, self._client, self._gpus)
+
+            if model_config.is_ensemble():
+                if len(models) > 1:
+                    raise TritonModelAnalyzerException(
+                        f'\nProfiling of multiple models is not supported for ensemble models'
+                    )
+
+                if self._config.run_config_search_mode != 'quick':
+                    raise TritonModelAnalyzerException(
+                        f'\nBrute search mode is not supported for ensemble models'
+                        '\nPlease use quick search mode')
```

Comment:
can you add the exact option they need to add to the error message? If they run into this I don't want them to have to look through docs to figure out how to do it.

Reply:
Done.